### DISCUSSION TITLE
Icon request: jump-to (arrow)

### DISCUSSION THREAD

**Discussion by PayBas** [FortAwesome/Font-Awesome#21202] [Icon Requests]
Handy for things like pagination, or anywhere else where you want to "skip" content by jumping directly to an anchor.

![jump-to](https://cloud.githubusercontent.com/assets/3015760/3117692/8f4334fa-e731-11e3-8d9c-3d7b47083a5d.png)

**Comment by hanakin** [comment]
+1

**Comment by dmland** [comment]
+1

**Comment by orlandumike** [comment]
+1, maybe the icon should represent more that you're going over something.
In my context, there is a bunch of steps to complete and some of them are optional, so we want a quick link to skip the step.

**Comment by pixsrv** [comment]
+1

**Comment by ozottin** [comment]
+1

**Comment by PeterBujuSerenergy** [comment]
+1

**Comment by kskiwmt** [comment]
+1

**Comment by ghost** [comment]
+1

**Comment by justi** [comment]
+1

**Comment by alexandervantrijffel** [comment]
+1!

**Comment by amitection** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by orthlieb** [comment]
+1
